framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -206,26 +233,21 @@ public void viewpointAdaptMethod(
      * @param newClassTree invocation of the constructor
      * @param constructorType type of the constructor; is side-effected by this method
      */
-    public void viewpointAdaptConstructor(
+    public void atConstructorInvocation(
             NewClassTree newClassTree, AnnotatedExecutableType constructorType) {
         if (!hasDependentAnnotations()) {
             return;
         }
-        List<? extends ExpressionTree> args = newClassTree.getArguments();
-        viewpointAdaptExecutable(newClassTree, constructorType, args);
+        atInvocation(newClassTree, constructorType);
     }
 
     /**
      * Viewpoint-adapts a method or constructor invocation.
      *
      * @param tree invocation of the method or constructor
      * @param methodType type of the method or constructor; is side-effected by this method
-     * @param argTrees the arguments to the method or constructor; subexpressions of {@code tree}
      */
-    private void viewpointAdaptExecutable(
-            ExpressionTree tree,
-            AnnotatedExecutableType methodType,
-            List<? extends ExpressionTree> argTrees) {
+    private void atInvocation(ExpressionTree tree, AnnotatedExecutableType methodType) {
         assert hasDependentAnnotations();
         Element methodElt = TreeUtils.elementFromUse(tree);
         // The annotations on `viewpointAdaptedType` will be copied to `methodType`.
```

Comment:
This variable name is confusing.  I think maybe it is not yet the viewpoint-adapted type, but by the end of the method it will be.  Please clarify.